### PR TITLE
Convert site to jekyll

### PR DESCRIPTION
**Pull request by wibblymat**
Had to make the files be Markdown to get the front-matter working. I did convert some of the HTML to markdown syntax, but HTML gets passed through anyway so I didn't do it all.

R: @s3ththompson

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    </tr>
+    <tr>
+      <td>binary encoding</td>
+      <td>fixed-width SIMD</td>
+      <td>linear memory >4GiB</td>
+    </tr>
+    <tr>
+      <td>textual format</td>
+      <td></td>
+      <td>... and <a href="https://github.com/WebAssembly/design/blob/master/FutureFeatures.md">more</td>
```
Comment (s3ththompson) [inline index.md]: looks like there was a bug in the original source here. can you close the link tag here?

File: index.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    </tr>
+    <tr>
+      <td>cross-browser support</td>
+      <td></td>
+      <td></a></td>
```
Comment (s3ththompson) [inline index.md]: delete `</a>`

File: index.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    </tr>
+    <tr>
+      <td>polyfill-able in JS</td>
+      <td></td>
+      <td></a></td>
```
Comment (s3ththompson) [inline index.md]: delete `</a>`

**Comment by s3ththompson** [comment]
closes #32

**Comment by wibblymat** [comment]
I've made those changes

**Comment by s3ththompson** [comment]
LGTM

**Pull request merged**